codemeta.json

```diff
@@ -0,0 +1,597 @@
+            "familyName": "Terrier",
+            "givenName": "R\u00e9gis"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Pamyra"
+            },
+            "familyName": "King",
+            "givenName": "Johannes"
+        },
+        {
+            "@id": "https://orcid.org/0000-0003-3274-4445",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Instituto de Astrof\u00edsica de Andaluc\u00eda - CSIC, Gta. de la Astronom\u00eda, 18008 Granada, Spain"
+            },
+            "familyName": "Ruiz",
+            "givenName": "Jos\u00e9 Enrique"
+        },
+        {
+            "@id": "https://orcid.org/0000-0002-8815-6530",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Max Planck Institut f\u00fcr Kernphysik, MPIK, Saupfercheckweg 1, 69117 Heidelberg, Germany"
+            },
+            "familyName": "Remy",
+            "givenName": "Quentin"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Jouvin",
+            "givenName": "L\u00e9a"
+        },
+        {
+            "@id": "https://orcid.org/0000-0002-9238-7163",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Universidad Complutense de Madrid EMFTEL, UCM, Plaza de la Ciencias, 28040 Madrid, Spain"
+            },
+            "familyName": "Sinha",
+            "givenName": "Atreyee"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Facebook, 1 Hacker Way, Menlo Park, CA 94025, US"
+            },
+            "familyName": "Wood",
+            "givenName": "Matthew"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "INAF/IASF, via Ugo La Malfa 153,  90146 Palermo, Italy"
+            },
+            "familyName": "Pintore",
+            "givenName": "Fabio"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Paz Arribas",
+            "givenName": "Manuel"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Max Planck Institut f\u00fcr Kernphysik, MPIK, Saupfercheckweg 1, 69117 Heidelberg, Germany"
+            },
+            "familyName": "Olivera",
+            "givenName": "Laura"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"
```

Comment:
the name of his affiliation is:
"Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris Cit\u00e9, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"

Reply:
I believe this should be changed in the `CITATION.cff` file (and probably in another PR)